MethodSwitch/src/methodswitch/MethodSwitch.java

```diff
@@ -0,0 +1,174 @@
+            System.out.println("FIZZBUZZ");
+        }
+        else if (fbNumber % 3 == 0){
+            System.out.println("FIZZ");
+        }
+        else if (fbNumber % 5 == 0){
+            System.out.println("BUZZ");
+        }
+        else{
+            System.out.println("Invalid number");
+        }
+    }
+    
+    /**
+     * Takes a user inputted word and uses a for loop with the charAt method
+     * to find the ascii value of each letter and then totals them in an
+     * accumulator.
+     */
+    public static void wordValue(){
+        String userWord;
+        int letter = 0;
+        int total = 0;
+        Scanner keyboard = new Scanner(System.in);
+        
+        System.out.println("Enter the word you want to see the value of:");
+        userWord = keyboard.nextLine();
+        
+        for(int i = 0; i < userWord.length(); i++){
+            letter = userWord.charAt(i);
+            System.out.println(letter);
+            total += letter;
+        }
+        System.out.println(total);
+    }
+    
+    /**
+     * Takes user inputted hours and wage and multiplies them to get a total 
+     * amount.
+     */
+    public static void salaryCalculator(){
+        double hours;
+        double wage;
+        double salary;
+        Scanner keyboard = new Scanner(System.in);
+        
+        System.out.println("Enter the amount of hours worked:");
+        hours = keyboard.nextDouble();
+        System.out.println("Enter the wage per hour:");
+        wage = keyboard.nextDouble();
+        salary = wage * hours;
+                
+        System.out.printf("The weekly salary is $%,.2f\n",salary);
+    }
+    
+    /**
+     * Takes a user inputted score to find the matching letter grade with if 
+     * else statements.
+     */
+    public static void checkGrade(){
+        double grade;
+        Scanner keyboard = new Scanner(System.in);
+
+        System.out.println("Enter your current score out of 100:");
+        grade = keyboard.nextDouble();
+                
+        if (grade >= 90){
+            System.out.println("You have an A.");
+        }
+        else if (grade >= 80){
+            System.out.println("You have a B.");
+        }
+        else if (grade >= 70){
+            System.out.println("You have a C.");
+        }
+        else if (grade >= 60){
+            System.out.println("You have a D.");
+        }
+        else{
+            System.out.println("You have an F.");
+        }
+    }
+    
+    /**
+     * Takes a user inputted response and ignores the case to either be equal
+     * to true or false and output the correct response with an invalid
+     * statement if it is neither.
+     */
+    public static void truthDare(){
+        String response;
+        Scanner keyboard = new Scanner(System.in);
+        
+        System.out.println("Enter 'Truth' or 'Dare':");
+        response = keyboard.next();
+                
+        if (response.equalsIgnoreCase("TRUTH")){
```

Comment:
These truth or dare literals are perfect opportunities to use constants within your function.

```java
final String TRUTH = "truth";
final String DARE = "dare";

if(response.equalsIgnoreCase(TRUTH)){}
if(response.equalsIgnoreCase(DARE)){}
```

No requirement to make this change just a way of thinking about it. I felt this `"TRUTH"` variable was almost screaming out like you wanted it to be a constant :+1: